### DISCUSSION TITLE
How can i show colum Phone and two icons Phone and SMS

### DISCUSSION THREAD

**Discussion by AliCod98** [filamentphp/filament#13494] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

Tables\Columns\TextColumn::make('Phone')
                    ->label('Phone'),
Tables\Columns\IconColumn::make('Phone')
                    ->label('Call')
                    ->icon('heroicon-m-phone')
                    ->url(fn ($record) => 'tel:' . $record->Phone, true),
Tables\Columns\IconColumn::make('Phone')
                    ->label('SMS')
                    ->icon('heroicon-m-envelope')
                    ->url(fn ($record) => 'sms:' . $record->Phone, true),
it's work but it show only this Tables\Columns\IconColumn::make('Phone')
                    ->label('SMS')
                    ->icon('heroicon-m-envelope')
                    ->url(fn ($record) => 'sms:' . $record->Phone, true), in the table

**Comment by leandrocfe** [comment]
I think because they have the same column name. Try do add different column ids or keys `->id('xx')` `->key('xx')`

**Reply by AliCod98** [reply]
IconColumn::id does not exist.

**Reply by AliCod98** [reply]
I fixed with different make('xx') 
Tables\Columns\IconColumn::make('Telephone')
                    ->label('')
                    ->icon('heroicon-m-phone')
                    ->url(fn ($record) => 'tel:' . $record->Telephone, true),
Tables\Columns\IconColumn::make('ref')
                    ->label('')
                    ->icon('heroicon-m-envelope')
                    ->url(fn ($record) => 'sms:' . $record->Telephone, true),
it's not a logical solution but it work for now until i find a solution

**Reply by leandrocfe** [reply]
Did you try ->key('x') ?
